tests/unit/libs/fetcher.client.js

```diff
@@ -39,42 +44,29 @@ describe('Client Fetcher', function () {
         before(function(){
             mockery.registerMock('./util/http.client', {
                 get: function (url, headers, config, done) {
-                    var urlObj = libUrl.parse(url);
-                    var pathname = urlObj.pathname;
-                    pathname = decodeURIComponent(pathname);
-                    pathname = pathname.split(';');
-                    expect(pathname.shift()).to.equal('/api/resource/' + resource);
-                    while(!!(pair = pathname.shift())) {
-                        var pair = pair.split('=');
-
-                        var k = pair[0],
-                            v = pair[1];
-
-                        //hacky because of array in querystring, but its fine
```

Comment:
@redonkulus was the hack to get around arrays not being serialized properly before...